pkg/iosrc/s3.go

```diff
@@ -18,17 +18,21 @@ var defaultS3Source = &s3Source{}
 var _ Source = defaultS3Source
 var _ ReplacerAble = defaultS3Source
 
+func init() {
+	defaultS3Source.Client = s3io.NewClient(nil)
```

Comment:
I believe you can move this into the declaration of `defaultS3Source` and remove the init.